expressEngine/lib/parser/parser.js

```diff
@@ -0,0 +1,20 @@
+export default (controller = null, functionName = '') => async (req, res) => {
```

Comment:
@NishantNaithani1997 Parser is not acting as a middleware here, please update it, use a json to validate all fields on the basis of request. Pass only required field to controller using next().